docs/getting-started.rst

```diff
@@ -161,74 +158,45 @@ Columns returned by the simulator
+| arbeitsl_v_beit_m       | Monthly amount employee unempl. insurance contrib. | float |
++-------------------------+----------------------------------------------------+-------+
+| ges_krankv_beit_m       | Monthly amount employee health insurance contrib.  | float |
++-------------------------+----------------------------------------------------+-------+
+| pflegev_beit_m          | Monthly amount of long term care insurance         | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_m         | Monthly amount of unemployment assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| rente_anspr_m           | Monthly amount of old-age pensions                 | float |
++-------------------------+----------------------------------------------------+-------+
+| abgelt_st               | Monthly capital income tax due, individual         | float |
++-------------------------+----------------------------------------------------+-------+
+| soli_st                 | Monthly solidarity surcharge due, individual       | float |
++-------------------------+----------------------------------------------------+-------+
+| kindergeld              | Monthly child Benefit, individual                  | float |
++-------------------------+----------------------------------------------------+-------+
+| eink_st                 | Monthly income Tax Due, individual                 | float |
++-------------------------+----------------------------------------------------+-------+
+| eink_st_tu              | Monthly income Tax Due, couple sum                 | float |
++-------------------------+----------------------------------------------------+-------+
+| unterhaltsvors_m        | Alimony advance payment, individual                | float |
++-------------------------+----------------------------------------------------+-------+
+| regelsatz_m             | Household socio-economic *need*, lump-sum          | float |
++-------------------------+----------------------------------------------------+-------+
+| kost_unterk_m           | Housing cost covered by social assistance          | float |
++-------------------------+----------------------------------------------------+-------+
+| kinderzuschlag_m        | Monthly additional child benefit, household sum    | float |
++-------------------------+----------------------------------------------------+-------+
+| wohngeld_m              | Monthly housing benefit, household sum             | float |
++-------------------------+----------------------------------------------------+-------+
+| arbeitsl_geld_2_m       | Monthly social assistance, household sum           | float |
++-------------------------+----------------------------------------------------+-------+
+| verfüg_eink_m           | Monthly disposable income, individual              | float |
```

Comment:
I would prefer to add a "b": `verfügb_eink`. Thoughts?

Reply:
Changed it!